templates/nextflow-aurora-mysql.yaml

```diff
@@ -65,6 +65,18 @@ Resources:
         ExcludeCharacters: '"@/\$`&'
         PasswordLength: 16
 
+  AuroraTowerUserSecret:
```

Comment:
would it make sense to change `Tower` to a more generic name in case nexflow change's it's DB user?  something like `AuroraNextflowUserSecret` would make it agnostic to a specific `Tower` username.  That way you can just pass in any username you want.

Reply:
Sure, I can name it something like `AuroraNextflowTowerDatabaseUserSecret`